pkg/stub/handler.go

```diff
@@ -678,6 +769,18 @@ func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
 			return nil
 		}
 
+		doit, err := h.ProcessManagementField(srcfg)
+		if err != nil {
+			// flush status update
+			h.sdkwrapper.Update(srcfg)
+			return err
+		}
+		if !doit {
+			// flush status update
+			h.sdkwrapper.Update(srcfg)
+			return nil
+		}
```

Comment:
combine these into a single if?  `(if !doit || err!=nil)`

Reply:
I am varying whether I return an error or not ... that is why the are not combined

Reply:
duh err is nil in the case where i return nil ... will change